guidelines/index.html

```diff
@@ -1573,14 +1573,14 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
+            <section class="guideline" id="dragging-movements"><div class="header-wrapper"><h4 id="x2-5-7-dragging-movements"><bdi class="secno">達成基準 2.5.7 </bdi>ドラッグ動作</h4><a class="self-link" href="#dragging-movements" aria-label="Permalink for Section 2.5.7"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html">Understanding Dragging Movements</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#dragging-movements">How to Meet Dragging Movements</a></div><p class="conformance-level">(レベル AA)</p>
 	<p class="change">[New]</p>
     
-  <p>All <a href="#dfn-functionality" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-functionality-6" title="利用者の操作により実現可能なプロセス及び結果。">functionality</a> that uses a <a href="#dfn-dragging-movements" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-dragging-movements-1" title="[New]">dragging movement</a> for operation can be achieved by a <a href="#dfn-single-pointer" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-single-pointer-3" title="シングルタップ・クリック、ダブルタップ・クリック、長押し、軌跡ベースのジェスチャなど、画面と一つの接点で動作するポインタ入力。">single pointer</a> without dragging, unless dragging is <a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-14" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">essential</a> or the functionality is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-2" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and not modified by the author.</p>
+  <p><a href="#dfn-dragging-movements" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-dragging-movements-1" title="[New]">ドラッグ動作</a>を用いて操作する<a href="#dfn-functionality" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-functionality-6" title="利用者の操作により実現可能なプロセス及び結果。">機能</a>はすべて、ドラッグなしの<a href="#dfn-single-pointer" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-single-pointer-3" title="シングルタップ・クリック、ダブルタップ・クリック、長押し、軌跡ベースのジェスチャなど、画面と一つの接点で動作するポインタ入力。">シングルポインタ</a>で完遂することができる。ただしドラッグが<a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-14" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">必要不可欠</a>である場合や、その機能が<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-2" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者が変更していない場合は除く。</p>
     <div class="note" role="note" id="issue-container-generatedID-36"><div role="heading" class="note-title marker" id="h-note-36" aria-level="5"><span>注記</span></div><p class="">この要件は、ポインタの動作を解釈するウェブコンテンツに適用される (ユーザエージェントや支援技術の操作に必要なアクションには適用されない)。</p></div>
```

Comment:
これは多分、既存の訳（どこ）から持ってきていると思いますが、「や」がいるので手を加えておきましょう…

> This requirement applies to web content that interprets pointer actions (i.e. this does not apply to actions that are required to operate the user agent or assistive technology).

```suggestion
    <div class="note" role="note" id="issue-container-generatedID-36"><div role="heading" class="note-title marker" id="h-note-36" aria-level="5"><span>注記</span></div><p class="">この要件は、ポインタの動作を解釈するウェブコンテンツに適用される (すなわち、これはユーザエージェント又は支援技術の操作に必要なアクションには適用されない)。</p></div>
```